decidim-accountability/spec/events/decidim/accountability/result_progress_updated_event_spec.rb

```diff
@@ -14,12 +14,17 @@
   let(:extra) { { proposal_id: proposal.id, progress: 95 } }
   let(:proposal_path) { resource_locator(proposal).path }
   let(:proposal_title) { translated(proposal.title) }
+  let(:email_subject) { "An update to #{translated resource.title} progress" }
+  let(:notification_title) { "The result <a href=\"#{resource_path}\">#{resource_title}</a>, which includes the proposal <a href=\"#{proposal_path}\">#{proposal_title}</a>, is now 95% complete." }
+  let(:email_outro) { "You have received this notification because you are following \"#{proposal_title}\", and this proposal is included in the result \"#{translated resource.title}\". You can stop receiving notifications following the previous link." }
+  let(:email_intro) { "The result \"#{translated resource.title}\", which includes the proposal \"#{proposal_title}\", is now 95% complete. You can see it from this page:" }
 
   before do
     resource.link_resources([proposal], "included_proposals")
   end
 
   it_behaves_like "a simple event", proposal_text: true
+  it_behaves_like "a simple event notification"
```

Comment:
Missing the email shared example: 
```suggestion
  it_behaves_like "a simple event email"
  it_behaves_like "a simple event notification"
```